backend/app/services/langgraph_chat/service.py

```
@@ -0,0 +1,364 @@
+    async def shutdown(self) -> None:
+        """Shutdown service and cleanup resources."""
+        if self.mcp_manager:
+            await self.mcp_manager.disconnect_all()
+
+    async def chat_completion(
+        self,
+        model: str,
+        messages: List[Dict[str, Any]],
+        stream: bool = False,
+        tools: Optional[List[Dict[str, Any]]] = None,
+        tool_choice: str = "auto",
```

Ruff (0.14.8)

Unused method argument: `tool_choice`

(ARG002)